PORTAL/jobs/__init__.py

```diff
@@ -70,20 +77,21 @@ def __init__(self, root='~/BENCH'):
         self.queue = _queue.JobQueueFS(self.jobs.requests)
 
     @property
-    def currentjob(self):
+    def currentjob(self) -> str:
         return self.jobs.requests.current
 
 
 class Jobs:
 
-    FS = PortalFS
+    FS: Type[_utils.FSTree] = PortalFS
 
-    def __init__(self, cfg, *, devmode=False):
+    def __init__(self, cfg: JobsConfig, *, devmode: bool = False):
         self._cfg = cfg
         self._devmode = devmode
         self._fs = self.FS(cfg.data_dir)
         self._workers = _workers.Workers.from_config(cfg)
         self._store = _pyperformance.FasterCPythonResults.from_remote()
+        self._queue: Optional[_queue.JobQueue] = None
```

Comment:
I meant without setting it to `None` there.  Then we could preserve the lazy property approach I was taking before (instead of the check-None-every-time approach).

Then again, if you think the lazy way isn't worth it then I'm fine with the `None` check in the property.  In that case, it may make sense to move the line back down into `__init__()`.  Sorry for the churn.